laravel/database/migrations/2021_04_06_152448_add_is_valid_column_to_option.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddIsValidColumnToOption extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('option', function (Blueprint $table) {
+            $table->integer('isValid');
```

Comment:
This is correct as the database exists today, but this column name should really be changed to `is_valid`.